optimize/backend/src/test/java/io/camunda/optimize/service/util/BpmnModelUtilTest.java

```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.optimize.service.util;
+
+import static io.camunda.optimize.util.ZeebeBpmnModels.END_EVENT;
+import static io.camunda.optimize.util.ZeebeBpmnModels.SERVICE_TASK;
+import static io.camunda.optimize.util.ZeebeBpmnModels.START_EVENT;
+import static io.camunda.optimize.util.ZeebeBpmnModels.USER_TASK;
+import static io.camunda.optimize.util.ZeebeBpmnModels.createSimpleServiceTaskProcess;
+import static io.camunda.optimize.util.ZeebeBpmnModels.createSimpleUserTaskProcess;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.optimize.dto.optimize.FlowNodeDataDto;
+import io.camunda.zeebe.model.bpmn.Bpmn;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+import org.camunda.bpm.model.bpmn.BpmnModelInstance;
+import org.junit.jupiter.api.Test;
+
+public class BpmnModelUtilTest {
+
+  public static final String CUSTOMER_ONBOARDING = "CustomerOnboarding";
+  public static final String END_EVENT_TYPE = "endEvent";
+  public static final String SERVICE_TASK_TYPE = "serviceTask";
+
+  public static final String SIMPLE_SERVICE_TASK_PROCESS =
+      Bpmn.convertToString(createSimpleServiceTaskProcess(CUSTOMER_ONBOARDING));
+
+  @Test
+  void shouldParseBpmnModel() {
+    final BpmnModelInstance modelInstance =
+        BpmnModelUtil.parseBpmnModel(SIMPLE_SERVICE_TASK_PROCESS);
+    assertThat(modelInstance).isNotNull();
+  }
```

Comment:
```suggestion
  void shouldParseBpmnModel() {
    // when
    final BpmnModelInstance modelInstance =
        BpmnModelUtil.parseBpmnModel(SIMPLE_SERVICE_TASK_PROCESS);
        
    // then
    assertThat(modelInstance).isNotNull();
  }
```

Conventionally, we comment the `given/when/then` steps in our tests for better readability

Reply:
fixed